helm_deploy/values-prod.yaml

```diff
@@ -2,7 +2,6 @@
 # Per environment values which override defaults in manage-offences-api/values.yaml
 
 generic-service:
-  replicaCount: 6
```

Comment:
this was introduced before caching (changed from 4 to 6 pods) in an attempt to improve performance, now that we have in-memory caching in place makes sense to revert back to the default of 4 pods